src/features/landing/ui/point-card/point-card.tsx

```
@@ -0,0 +1,19 @@
+import { POINT_CARD_ITEMS } from "../../config/point-card.constant";
+
+import * as styles from "./point-card.css";
+
+export const PointCard = () => {
+  return (
+    <div className={styles.layout}>
+      {POINT_CARD_ITEMS.map((point) => (
+        <section className={styles.container}>
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)